{"status": "broken", "description": "Activating the pick action test and checking whether it was successful.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "pick_test"}, {"name": "subSuite", "value": "TestPickAction"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "18551-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.pick_test"}], "stop": 1607982598664, "uuid": "ad5c510b-b068-4e47-bea9-07da9a2f5a71", "historyId": "dfe1c67d4188a9f3c1ffbc72d27b2e12", "name": "test_verification_of_pick_action", "start": 1607982587720, "fullName": "tests.pick_test.TestPickAction#test_verification_of_pick_action", "statusDetails": {"message": "RuntimeError: No Trajectory execution capability available.", "trace": "self = <tests.pick_test.TestPickAction instance at 0x7f87865e3cf8>\n\n    def test_verification_of_pick_action(self):\n        \"\"\"Activating the pick action test and checking whether it was successful.\n        \"\"\"\n        mo = Model('glass')\n        data_logger('logger/logs/pick_action_start')\n        hx,hy,hz = mo.lucy_pos()[0],mo.lucy_pos()[1],mo.lucy_pos()[2]\n        # result = picker_client(hx+0.65, hy, 0.5, 0.0, 0.0, 0.0)\n        result = MoveItPickAndPlace( pick_x = hx+0.75, pick_y = hy, pick_z = 0.55,\n>                                    place_x = hx+0.7, place_y = hy, place_z = 0.76)\n\ntests/pick_test.py:79: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/action_client/pick_place_client.py:30: in __init__\n    gripper = moveit_commander.MoveGroupCommander(\"gripper\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <moveit_commander.move_group.MoveGroupCommander object at 0x7f8787501dd0>\nname = 'gripper', robot_description = 'robot_description', ns = ''\n\n    def __init__(self, name, robot_description=\"robot_description\", ns=\"\"):\n        \"\"\" Specify the group name for which to construct this commander instance. Throws an exception if there is an initialization error. \"\"\"\n>       self._g = _moveit_move_group_interface.MoveGroupInterface(name, robot_description, ns)\nE       RuntimeError: No Trajectory execution capability available.\n\n/opt/ros/kinetic/lib/python2.7/dist-packages/moveit_commander/move_group.py:52: RuntimeError"}, "attachments": [{"source": "fd350e4c-88ba-46d6-9406-93e9505638c2-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "6c03267e-7fd4-44b8-af8f-f2cf71cdcc42-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "404969ff-473e-46c0-8a8a-456f1305fd5e-attachment.txt", "type": "text/plain", "name": "stderr"}]}